src/Tvheadend.cpp

```diff
@@ -664,7 +664,8 @@ PVR_ERROR CTvheadend::GetTimerTypes ( PVR_TIMER_TYPE types[], int *size )
   }
 
   /* PVR_Timer.iPreventDuplicateEpisodes values and presentation.*/
-  static std::vector< std::pair<int, std::string> > deDupValues;
+  std::vector< std::pair<int, std::string> > deDupValues;
```

Comment:
I commented on this back when this was introduced but @Glenn-1990 said this method was only ever called once or something like that.